packages/style/README.md

```diff
@@ -23,12 +23,6 @@ or if you want to import it in your JS files
 import "monday-ui-style/dist/index.min.css";
 ```
 
-**Icons**: We expose each icon's SVG from the dist files so you can use them in your app.
-
-```javascript
-import IconName from "monday-ui-style/src/Icons/IconName.svg";
-```
-
```

Comment:
We need to mention it in the changelog as well as this is breaking, and we'll have to release a major for this one (which is legit)

Reply:
let's discuss it and accumulate more stuff for a major
we should also take into consideration people who consume us **inside monday** through unpkg. they need to set themselves to a v0 tag instead of latest (which is our responsibility to add this kind of tag to a version of `style`)

Reply:
btw, we probably need to add `BREAKING` or `chore(icons)!:...`, so Lerna would know to release a major for `style`

Reply:
As discussed I moved the changes of style to a different [pr](https://github.com/mondaycom/vibe/pull/2449)